lib/ui/login/login_screen.dart

```diff
@@ -48,36 +70,160 @@ class _LoginScreenState extends State<LoginScreen>
       }
     });
 
+    _logoPositionAnimationController.addStatusListener((status) {
+      if (status == AnimationStatus.completed) {
+        _overlayOpacityAnimationController.forward();
+      }
+    });
+
     _logoOpacityAnimationController.forward();
   }
 
-  Stack _buildLoginScreen() => Stack(
+  TextStyle _textFieldHintTextStyle() => TextStyle(
+        color: Colors.white.withOpacity(0.3),
+        fontFamily: FontFamily.neuzeit,
+        fontSize: AppDimensions.textSize17,
+        fontWeight: FontWeight.w400,
+      );
+
+  TextStyle? _textFieldTextStyle(BuildContext context) =>
+      Theme.of(context).textTheme.bodySmall;
+
+  TextStyle _loginButtonTextStyle() => const TextStyle(
+        color: AppColors.blackRussian,
+        fontFamily: FontFamily.neuzeit,
+        fontSize: AppDimensions.textSize17,
+        fontWeight: FontWeight.w800,
+      );
+
+  InputDecoration _textFieldInputDecoration(String placeholderText) =>
+      InputDecoration(
+        fillColor: Colors.white24,
+        filled: true,
+        hintText: placeholderText,
+        hintStyle: _textFieldHintTextStyle(),
+        contentPadding: const EdgeInsets.symmetric(
+          vertical: AppDimensions.spacing18,
+          horizontal: AppDimensions.spacing12,
+        ),
+        border: OutlineInputBorder(
+          borderSide: BorderSide.none,
+          borderRadius: BorderRadius.circular(AppDimensions.radius12),
+        ),
+      );
```

Comment:
Shall we define this inside `app_theme.dart` as `inputDecorationTheme`?

Then it can be applied to all the `TextInput`'s in our app

Inside `app_theme.dart`

```
inputDecorationTheme: InputDecorationTheme(
  fillColor: Colors.white24,
  filled: true,
  hintStyle: TextStyle(
    color: Colors.white.withOpacity(0.3),
    fontSize: AppDimensions.textSize17,
    fontWeight: FontWeight.w400,
  ),
  contentPadding: const EdgeInsets.symmetric(
    vertical: AppDimensions.spacing18,
    horizontal: AppDimensions.spacing12,
  ),
  border: OutlineInputBorder(
    borderSide: BorderSide.none,
    borderRadius: BorderRadius.circular(AppDimensions.radius12),
  ),
),
```

**Reference**: https://www.flutterbeads.com/change-hint-text-color-in-flutter/

Then, inside `login_screen.dart`, something like this? (there might be a better way)

```
decoration: const InputDecoration()
  .applyDefaults(Theme.of(context).inputDecorationTheme)
  .copyWith(
    hintText: isEmail
        ? AppLocalizations.of(context)?.email ?? 'Email'
        : AppLocalizations.of(context)?.password ?? 'Password',
),
```

Oh, one more thing... For a white cursor in the text input, we should add this to `add_theme.dart`:

```
 textSelectionTheme: const TextSelectionThemeData(
    selectionColor: Colors.white
  ),
```

**Reference**: https://stackoverflow.com/questions/65897342/how-to-change-textfield-cursor-color-in-flutter

cc @luongvo